packages/dashboard/src/components/DashboardFilters/DashboardFilters.tsx

```diff
@@ -91,18 +91,23 @@ const DashboardFilters: React.FC<DashboardFilterProps> = ({
           // Data Filter
           filter.values?.forEach((filterOption, index) => {
             const labeledOpt = filter.labels && filter.labels[filterOption]
-            values.push(
+            const shouldUnshift = (filterOption || labeledOpt) === filter.resetLabel
+            // add label to the front of list if it matches with reset label
+            values.splice(
+              shouldUnshift ? 0 : values.length,
+              0,

```

Comment:
I'm having a hard time reading all this. could you change shouldUnshift to "isResetLabel" and then change thse values.splice to and if else statement? 

if(isResetLabel)

// unshift it

else 

// do what the code was doing before